src/client/actions/MessageReactionAdd.js

```diff
@@ -28,6 +28,10 @@ class MessageReactionAdd extends Action {
 
     // Verify reaction
     if (message.partial && !this.client.options.partials.includes(PartialTypes.REACTION)) return false;
+    const existing =
+      message.reactions.cache.has(data.emoji.name) &&
+      message.reactions.cache.get(data.emoji.name).users.cache.get(user.id);
```

Comment:
you need to account for custom emojis by using the ID

Reply:
you're right, fixed in 64f14cfcb6c33ad9ff4aac27862050a737a958d0